### PR TITLE
Agreeing on how to agree

### PR DESCRIPTION
**Pull request by Mitzi-Laszlo**
Solid has grown. While in a small team it was easy to simply talk to each other and come to decisions together, in our larger team this is now can be a little unwieldy. Without creating fatty tissue process for no reason and making sure Solid stays mean and lean, I want to open up a conversation about how to come to decisions together to deliver a cohesive Solid Community Voice. How do we agree to agree? What is the process of making a suggestion and coming to a legitimate conclusion together. How do we overcome differences of opinion to make sure we continue moving forward? 

Let's take what has worked with us, there's no need to reinvent the wheel. Let's write it down so that it is open to anyone who wants to join. Our Solid Community has expanded to include non-developers as well as developers. Non-developers may not be as familiar with the tools we are using so it is important to include resources to quickly explain the necessary parts of the tools we are using to make sure all voices are heard regardless of background.  

On top of the suggesting to the readme on processes, I would like to invite a conversation about label and their descriptions. https://github.com/solid/solid/issues/labels 

Also, who should get which admin privileges and why? https://github.com/orgs/solid/teams What responsibilities do people have per team or admin privilege? 

Lastly, how do we as Solid coordinate with other parties looking to contribute to Solid? For example, pod providers, webid providers, developer tool providers, and Solid app developers.

This is just a suggestion please do feel free to build on it, I am not stuck to this suggestion at all. What matters is that we co-create a process that we can all feel comfortable with that allows our growing community to make decisions with legitimacy.

### CODE REVIEW

**Comment by Mitzi-Laszlo** [comment]
https://github.com/solid/community/issues/7 this could also possibly be instrumental in working towards an agreement comfortably

File: README.md [hunk 1]
```diff
@@ -32,16 +32,14 @@ Solid has a [code of conduct](code-of-conduct.md) which all community members mu
...
-Step 2. Consensus
-Community members can share ideas and propose solutions on each pull request or issue. 
+Step 2. Inviting a Conversation around your Suggestion to Find Consensus
+Let others know about your suggestion by posting it on solid/suggestions. Solid/suggestions is not a place to talk about the suggestion, this happens in the pull request or issue itself. Leave the pull request or issue open for two weeks to give others in the Solid community the opporunity to share ideas and propose solutions. 
 
-Step 3. Merge
-The relevant project release manager will judge and implement the merge. If there is a difference of opinion about a merge between the contributor and the relevant project release manager, they will first talk to try and find a compromise. When a compromise cannot be found the community leader will make the final judgement.
-
-If you are working on Solid let others know your processes.
+Step 4. Coming to a Conclusion through Compromise
```
Comment (RubenVerborgh) [inline README.md]: 3?

**Comment by konobi** [comment]
Here's a list of resources around decentralized organizing that may be useful to examine to see if there are patterns that may work for this project. It may even be helpful to get an idea of good ways of conveying the decided upon methods.

https://hackmd.io/s/Skh_dXNbE

**Comment by mikeadams1** [comment]
Can you define admin? can you define admin privileges? what are the qualifications one must meet to obtain admin, or admin privileges? and who authorizes the same? is there an application process?

**Comment by mikeadams1** [comment]
I think a good suggestion would be to create a place where Solid Community can receive weekly updates on any milestones, progress, goals. This would create a more cohesive union between Inrupt and the Solid Community of developers and non-developers. Time is money and therefore, some developers and non-developers may want to hold off investing too much time until Solid is at a point where there efforts wont be undone by changes they did not know were forthcoming.

**Comment by csarven** [comment]
@mikeadams1 I think "admin" is in context of GitHub and the repositories - general housekeeping, and also aligned with community's rules/conducts/expectations. As for qualifications, I'd say that having a good understanding of the Solid project and the repositories in question would be a good start, as well as having a positive track record and involvement. For all intents and purposes, that's evident or visible from the individual's prior output eg. contributions in chats, code, issues, in person etc. Needless to say, there is also an expectation that admins can dedicate time to attend the duties within a reasonable amount of time, so that they don't pose as a bottle-neck. It is neither expected that an admin would act counter to the consensus reached by the community or pose friction. It is definitely not a badge. Just as admin privileges can be given, they can be taken away.

IMHO.

**Comment by RubenVerborgh** [comment]
> I think a good suggestion would be to create a place where Solid Community can receive weekly updates on any milestones, progress, goals.

I am looking at setting up a call schedule within the open Solid W3C Community group.

File: README.md (hunk 1 above)
Comment (csarven) [reply README.md]: Profit.

**Comment by mikeadams1** [comment]
@csarven thank you for your response, I don't believe it could have been stated any better; "It is neither expected that an admin would act counter to the consensus reached by the community or pose friction."

@RubenVerborgh That would be great, as Solid continues to evolve and grow, a call schedule would be very helpful.

Because of this topic, "Agreeing on how to agree #27, am agreeing to agree with the Community and am donating https://inrupt.community domain to the Community.

File: README.md [hunk 2]
```diff
@@ -32,16 +32,14 @@ Solid has a [code of conduct](code-of-conduct.md) which all community members mu
...
-Step 1. Submission
-Submit a GitHub pull request if you would like to make a specific change to code or text or a [issue](https://github.com/solid/community/blob/master/issue-template.md) if you would like to raise a general point. If you are working on an issue or pull request assign yourself to that issue to let everyone you are working on it.
+Step 1. Making a Suggestion
+Submit a GitHub pull request if you would like to make a specific change to code or text or a [issue](https://github.com/solid/community/blob/master/issue-template.md) if you would like to raise a general point. Assign the appropriate labels by listing them in the optional extend description. 
 
-Step 2. Consensus
-Community members can share ideas and propose solutions on each pull request or issue. 
+Step 2. Inviting a Conversation around your Suggestion to Find Consensus
+Let others know about your suggestion by posting it on solid/suggestions. Solid/suggestions is not a place to talk about the suggestion, this happens in the pull request or issue itself. Leave the pull request or issue open for two weeks to give others in the Solid community the opporunity to share ideas and propose solutions.
```
Comment (ericprud) [inline README.md]: + Link to solid/suggestions?

File: README.md [hunk 3]
```diff
@@ -32,16 +32,14 @@ Solid has a [code of conduct](code-of-conduct.md) which all community members mu
...
 
-Step 3. Merge
-The relevant project release manager will judge and implement the merge. If there is a difference of opinion about a merge between the contributor and the relevant project release manager, they will first talk to try and find a compromise. When a compromise cannot be found the community leader will make the final judgement.
-
-If you are working on Solid let others know your processes.
+Step 4. Coming to a Conclusion through Compromise  
+Two weeks after the opening of the pull request or issue, members of the Solid Team are responsible for confirming the final decision, assiging action points to those responsible, and merging pull requests and issues. If there is a difference of opinion about a merge, parties are encoursed to talk to find a compromise. As a last resort, when a compromise cannot be found the community leader will make the final judgement.
```
Comment (ericprud) [inline README.md]: I hate to add still more adipose tissue, but what's a submitter to do if no one responds to their PR or issue within two weeks? The bulk of these will probably not be issues where no compromise was reachable, but instead that not enough people invested the time to understand the problem or make a persuasive argument to the commentor that the status quo is sufficient.

In principle, some sort of automagic process should notify someone that these are getting stale, but I suspect a **\[hey, it's time!\]** button will reduce commentor frustration.

**Comment by Mitzi-Laszlo** [comment]
I'm thinking that two week turn arounds for the incorporation of suggestion is a bit slow and propose a one week turnaround to make Solid more agile. Any objections?

**Comment by RubenVerborgh** [comment]
I think two weeks are good, if there is no particular urgency. One week makes things easy to miss.

File: README.md (hunk 3 above)
Comment (Mitzi-Laszlo) [reply README.md]: Added a sentence about what if the issue or pull request is open for more than two weeks...

**Comment by Mitzi-Laszlo** [comment]
https://github.com/solid/community/pull/41

**Pull request closed without merging**